src/index.js

```diff
@@ -439,7 +439,8 @@ class ReactTooltip extends Component {
       {'type-warning': this.state.type === 'warning'},
       {'type-error': this.state.type === 'error'},
       {'type-info': this.state.type === 'info'},
-      {'type-light': this.state.type === 'light'}
+      {'type-light': this.state.type === 'light'},
+      {'type-custom': !this.state.type}
```

Comment:
so 'custom' ends up being the same as not specifying a type, correct?

To merge this, I'd like to see an example added to illustrate how to use it.